### ISSUE TITLE
Vietnamese Keyboard bug

### ISSUE THREAD

**Issue by PhuongBNg** [Genymobile/scrcpy#3501]
- [V] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [V] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows
 - scrcpy version: 1.24
 - installation method: Windows released
 - device model: Pixel 4A, Pixel 4XL, Pixel 3, Pixel 4
 - Android version: 12-13

**Describe the bug**
I cant type any Vietnamese character with SCRCPY. I still able to type character with 1 special symbol but not with two. I am using EVkey or Unikey.

For Example:

"Hà" "Hù" "Hò" are type-able.

BUT

"Hồ", "Huấn", "Tuần,"... it just deleted everything. --- To type Telex with (EVkey & Unikey)

ồ = o o f
ấ = a a s
ầ = a a f

```
WARN: Could not inject char u+1ea5
```

Please do not post screenshots of your terminal, just post the content as text instead.

**Comment by rom1v** [comment]
Try `scrcpy --prefer-text`. But accented characters support is inherently limited via input injection: https://blog.rom1v.com/2018/03/introducing-scrcpy/#handle-accented-characters

You might try [HID keyboard](https://github.com/Genymobile/scrcpy#physical-keyboard-simulation-hid).

**Comment by PhuongBNg** [comment]
> > I tried HID but didnt work ether
> 
> Did you configure the keyboard layout as explained in the README: https://github.com/Genymobile/scrcpy#physical-keyboard-simulation-hid ?

Yes but our program act differently, it more like replace old char with new char when needed.

**Comment by PhuongBNg** [comment]
> > I tried HID but didnt work ether
> 
> Did you configure the keyboard layout as explained in the README: https://github.com/Genymobile/scrcpy#physical-keyboard-simulation-hid ?

More or less, it worth without HID now, thank you.

**Comment by quyleanh** [comment]
> Sadily didnt work, I tried HID but didnt work ether, but I found another way already Thanks, but install keyboard on the phone instead of using Gboard

Could you tell me your work around?
What OS you are using to remote Android device?
Thank you.

**Comment by PhuongBNg** [comment]
Just install any keyboard and uninstall Gboard, on any devices (Tested on
multiple different devices like S10, S8, Samsung 4A,...). Use keyboard like
laban
